setup.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/python3
```

Comment:
Can we remove this hash bang? Would affect installing using python2

Reply:
Sure, gonna change it.